homeassistant/components/hassio/handler.py

```diff
@@ -617,3 +608,9 @@ async def send_command(
             _LOGGER.error("Client error on %s request %s", command, err)
 
         raise HassioAPIError
+
+
+def get_supervisor_client(hass: HomeAssistant) -> SupervisorClient:
+    """Return supervisor client."""
+    hassio: HassIO = hass.data[DOMAIN]
+    return hassio.client
```

Comment:
You can use HassKey in a future PR to add typing to the `hass.data` so you can avoid this type overwrite (if wanted)

Reply:
Wasn't aware of that, that would be nice. It would definitely have to be a follow-up PR though as doing that would require a significant refactor to this integration looking at how much we use that currently.